### ISSUE TITLE
TPU num_processes indentation error

### ISSUE THREAD

**Issue by JTT94** [huggingface/accelerate#34]
In file https://github.com/huggingface/accelerate/blob/main/src/accelerate/commands/config/cluster.py there is an indentation error, leading to `num_processes` being undefined when using TPU.

```
if distributed_type == DistributedType.TPU:
        main_training_function = _ask_field(
            "What is the name of the function in your script that should be launched in all parallel scripts? [main]: ",
            default="main",
        )
    else:
        main_training_function = "main"

        num_processes = _ask_field(
            "How many processes in total will you use? [1]: ",
            lambda x: int(x),
            default=1,
            error_message="Please enter an integer.",
        )

        if distributed_type != DistributedType.TPU:
            fp16 = _ask_field(
                "Do you wish to use FP16 (mixed precision)? [yes/NO]: ",
                _convert_yes_no_to_bool,
                default=False,
                error_message="Please enter yes or no.",
            )
        else:
            fp16 = False
```

**Comment by JTT94** [comment]
Can you point me to the contribution guidelines and I can submit a fix?

**Comment by sgugger** [comment]
Oh yes, that's a bad one. Thanks for pointing it out! Feel free to open a PR, the main guideline is to run `make style` on your branch before pushing to have the auto-formatting applied (though I doubt it will change much in this case).

**Comment by JTT94** [comment]
Great, all done. https://github.com/huggingface/accelerate/pull/35

**Comment by sgugger** [comment]
Fixed by #35
